west.yml

```diff
@@ -21,6 +21,8 @@ manifest:
   remotes:
     - name: upstream
       url-base: https://github.com/zephyrproject-rtos
+    - name: yonsch
+      url-base: https://github.com/yonsch
```

Comment:
I think this is not the correct way to add new modules / HAL and for sure `yonsch` is not a proper name for the remote :)

Reply:
Sure, that's just to get the CI to pass